lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -726,7 +726,9 @@ def set_http_request(record, entry)
       'A' => 'ALERT',
       # other misc. translations.
       'ERR' => 'ERROR',
-      'F' => 'CRITICAL'
+      'F' => 'CRITICAL',
+      'CRIT' => 'CRITICAL',
```

Comment:
Want to move these into their own section above "misc translations" and label that section "nginx levels"?